### ISSUE TITLE
Html::root_element panics if the document has a Doctype node

### ISSUE THREAD

**Issue by kl** [programble/scraper#17]
This code panics (has a doctype):
```rust
let url = "https://stackoverflow.com/";
let html = reqwest::get(url).unwrap().text().unwrap();
let doc = Html::parse_document(&html);
doc.root_element();
```
while this does not (does not have doctype):
```rust
let url = "https://news.ycombinator.com/";
let html = reqwest::get(url).unwrap().text().unwrap();
let doc = Html::parse_document(&html);
doc.root_element();
```

After taking a quick look I think the problem is that root_element() assumes that the first element is always Element, so it panics when it is Doctype.

**Comment by kl** [comment]
Actually, it also panics if there are comments before the html node like here: https://www.smithsonianmag.com/science-nature/old-world-high-tech-141284744/
So maybe it can be solved with something like this?
```rust
    /// Returns the root `<html>` element.
    pub fn root_element(&self) -> ElementRef {
        let root_node = self
            .tree
            .root()
            .children()
            .find(|child| child.value().is_element())
            .expect("html node missing");
        ElementRef::wrap(root_node).unwrap()
    }
```

**Comment by programble** [comment]
released 0.9.1
